webservices/server-integration/src/main/java/org/jboss/as/webservices/deployers/AspectDeploymentProcessor.java

```diff
@@ -68,6 +74,8 @@ public void deploy(final DeploymentPhaseContext phaseContext) throws DeploymentU
             WSLogger.ROOT_LOGGER.tracef("%s start: %s", aspect, unit.getName());
             ClassLoader origClassLoader = WildFlySecurityManager.getCurrentContextClassLoaderPrivileged();
             try {
+                dep.setProperty("isHttpsOnly", isHttpsOnly(phaseContext, unit));
+
```

Comment:
@bstansberry @darranl As suggested by @jimma, I replaced the boolean attachment with a property. Looks cleaner and better.